app/views/home/index.html.slim

```diff
@@ -12,15 +12,15 @@
     - else
       = content_tag :div, msg, :id => "flash_#{name}"
 
-#myCarousel.carousel.slide data-ride="carousel" data-interval="5000"
-  ol class="carousel-indicators"
-    li data-target="#myCarousel" data-slide-to="0" class="active"
-    li data-target="#myCarousel" data-slide-to="1"
-    li data-target="#myCarousel" data-slide-to="2"
+#myCarousel.carousel.pt-3.slide[data-bs-ride="carousel"]
+  .carousel-indicators
+    button.active[type="button" data-bs-target="#myCarousel" data-bs-slide-to="0" aria-current="true" aria-label="Slide 1"]
+    button[type="button" data-bs-target="#myCarousel" data-bs-slide-to="1" aria-label="Slide 2"]
+    button[type="button" data-bs-target="#myCarousel" data-bs-slide-to="2" aria-label="Slide 3"]
   .carousel-inner
-    .item.active
-      = image_tag "banner1.jpg", style: "width:100%"
-      .carousel-caption
+    .carousel-item.active
+      = image_tag('banner1.jpg', class: 'd-block w-100', alt: '...')
```

Comment:
Should we add a meaningful alt text here?

Reply:
Done